src/v2/view-builder/views/custom-app/ChallengeCustomAppResendPushView.js

```diff
@@ -0,0 +1,43 @@
+import { loc, createCallout } from 'okta';
+import { BaseForm } from '../../internals';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'custom-app-verify-resend-push',
+
+    title() {
+      return loc('oie.verify.custom_app.title', 'login', [this.options.appState.getAuthenticatorDisplayName()]);
+    },
+
+    save() {
+      return loc('oie.custom_app.push.resend', 'login');
+    },
+
+    showMessages() {
+      // override showMessages to display error in cases like reject push
+      // or timeout. Borrowed this logic from TerminalView.
+      const messagesObjs = this.options.appState.get('messages');
```

Comment:
why not just update the `BaseForm` and just get rid of this redundancy ?

Reply:
For the same reason @kamaldeepcheema-okta  dint want to add it to the baseForm. It might bring in unexpected changes. We can add it as a separate commit in case we want to revert it.